assets/src/edit-story/elements/index.js

```diff
@@ -65,3 +65,14 @@ export const elementTypes = [
 
 export const getDefinitionForType = (type) =>
   elementTypes.find((el) => el.type === type);
+
+export const getTypeFromResource = (resource) => {
```

Comment:
If we confirm that resource shouldn't participate in dropping, then I think `element.type == resource.type`.